packages/next/telemetry/events/version.ts

```diff
@@ -21,13 +47,26 @@ export function eventVersion(
     {
       eventName: EVENT_VERSION,
       payload: {
+        ...event,
         nextVersion: process.env.__NEXT_VERSION,
         nodeVersion: process.version,
-        cliCommand: event.cliCommand,
-        isSrcDir: event.isSrcDir,
-        hasNowJson: event.hasNowJson,
-        isCustomServer: event.isCustomServer,
-      } as EventCliSessionStarted,
+        target: userConfig.target || null,
+        hasCustomWebpack: typeof userConfig.webpack === 'function',
+        hasCustomWebpackDev:
+          typeof userConfig.webpackDevMiddleware === 'function',
+        hasAssetPrefix: userConfig.assetPrefix?.length > 0,
+        hasCustomBuildId: typeof userConfig.generateBuildId === 'function',
+        hasDistDir: userConfig.distDir ? userConfig.distDir !== '.next' : false,
+        hasRuntimeConfig:
+          Object.keys(userConfig.publicRuntimeConfig ?? {}).length > 0 ||
+          Object.keys(userConfig.serverRuntimeConfig ?? {}).length > 0,
+        hasReactStrictMode: !!userConfig.reactStrictMode,
+        hasMdxPages:
+          Array.isArray(userConfig.pageExtensions) &&
+          userConfig.pageExtensions.includes('mdx'),
```

Comment:
Why not include all pageExtensions 🤔

Reply:
@timneutkens in our 200+ examples `mdx` and `md` (which I forgot to add, will change) were the only custom extensions, but that could be done, I didn't know of any other cases.

Reply:
@lfades people changing the extensions to something we're not aware of would be a good one.